src/main/resources/static/css/buttonOverrides.css

```diff
@@ -1,3 +1,25 @@
+.button_disabled
+{
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the opening curly brace to the previous line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=css%3Aformatting)